metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -0,0 +1,148 @@
+  | '@opentelemetry/instrumentation-graphql'
+  | '@opentelemetry/instrumentation-grpc'
+  | '@opentelemetry/instrumentation-koa'
+  | '@opentelemetry/instrumentation-ioredis'
+  | '@opentelemetry/instrumentation-express'
+  // | '@opentelemetry/instrumentation-mongodb'
+  // | '@opentelemetry/instrumentation-mysql'
+  | '@opentelemetry/instrumentation-pg';
+// | '@opentelemetry/instrumentation-redis'
+
+type InstrumentationConfigMap = {
+  '@opentelemetry/instrumentation-dns'?: DnsInstrumentationConfig;
+  '@opentelemetry/instrumentation-express'?: ExpressInstrumentationConfig;
+  '@opentelemetry/instrumentation-http'?: HttpInstrumentationConfig;
+  '@opentelemetry/instrumentation-graphql'?: GraphQLInstrumentationConfig &
+    InstrumentationConfig;
+  '@opentelemetry/instrumentation-grpc'?: GrpcInstrumentationConfig;
+  '@opentelemetry/instrumentation-koa'?: InstrumentationConfig;
+  '@opentelemetry/instrumentation-ioredis'?: IORedisInstrumentationConfig;
+  // '@opentelemetry/instrumentation-mongodb'?: MongoInstrumentationConfig,
+  // '@opentelemetry/instrumentation-mysql'?: MysqlInstrumentationConfig,
+  '@opentelemetry/instrumentation-pg'?: PgInstrumentationConfig;
+  // '@opentelemetry/instrumentation-redis'?: RedisInstrumentationConfig,
+};
+
+const InstrumentationMap: Record<InstrumentationMapKeys, any> = {
+  '@opentelemetry/instrumentation-dns': DnsInstrumentation,
+  '@opentelemetry/instrumentation-express': ExpressInstrumentation,
+  '@opentelemetry/instrumentation-http': HttpInstrumentation,
+  '@opentelemetry/instrumentation-graphql': GraphQLInstrumentation,
+  '@opentelemetry/instrumentation-grpc': GrpcInstrumentation,
+  '@opentelemetry/instrumentation-koa': KoaInstrumentation,
+  '@opentelemetry/instrumentation-ioredis': IORedisInstrumentation,
+  // '@opentelemetry/instrumentation-mongodb': MongoDBInstrumentation,
+  // '@opentelemetry/instrumentation-mysql': MySQLInstrumentation,
+  '@opentelemetry/instrumentation-pg': PgInstrumentation,
+  // '@opentelemetry/instrumentation-redis': RedisInstrumentation,
+};
+
+const defaultConfigs: InstrumentationConfigMap = {
+  '@opentelemetry/instrumentation-dns': { enabled: true },
+  '@opentelemetry/instrumentation-express': { enabled: true },
+  '@opentelemetry/instrumentation-http': { enabled: true },
+  '@opentelemetry/instrumentation-graphql': { enabled: true },
+  '@opentelemetry/instrumentation-grpc': { enabled: true },
+  '@opentelemetry/instrumentation-koa': { enabled: true },
+  '@opentelemetry/instrumentation-ioredis': { enabled: true },
+  // '@opentelemetry/instrumentation-mongodb': { enabled: true },
+  // '@opentelemetry/instrumentation-mysql': { enabled: true },
+  '@opentelemetry/instrumentation-pg': { enabled: true },
+  // '@opentelemetry/instrumentation-redis': { enabled: true }
+};
+
+export function getNodeAutoInstrumentations(
+  inputConfigs: InstrumentationConfigMap = {}
+): Instrumentation[] {
+  const configs: InstrumentationConfigMap = Object.assign(
```

Comment:
If you implement my suggestion, this will need to be built by looping over `Object.keys(InstrumentationMap)`